src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -134,7 +134,11 @@ public IRowToRowMapper GetRowToRowMapper(Schema inputSchema)
             return (IRowToRowMapper)Scorer.ApplyToData(Host, new EmptyDataView(Host, inputSchema));
         }
 
-        protected void SaveModel(ModelSaveContext ctx)
+        void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);
+
+        private protected abstract void SaveModel(ModelSaveContext ctx);
+
+        protected void SaveModelCore(ModelSaveContext ctx)
```

Comment:
>SaveModelCore [](start = 23, length = 13)

funny thing, this method get called only in one of three classes derived from PredictionTransformerBase

Reply:
Yes I had noticed that and found it quite strange!

---
In reply to: [255240879](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255240879) [](ancestors = 255240879)